tty/imx6ull-uart/imx6ull-uart.c

```diff
@@ -403,45 +655,59 @@ int main(int argc, char **argv)
 	if (portCreate(&port) != EOK)
 		return 2;
 
+	uart.rx_sw_fifo = malloc(sizeof(lf_fifo_t) + RX_SW_FIFO_SIZE * sizeof(uart.rx_sw_fifo->data[0]));
```

Comment:
All of malloc size arguments are const, this operation is always performed, imho let's avoid malloc if possible (it is possible here)